Modules/_curses_panel.c

```diff
@@ -671,8 +650,30 @@ PyInit__curses_panel(void)
     Py_INCREF(get_curses_panelstate(m)->PyCursesPanel_Type);
     PyModule_AddObject(m, "panel",
```

Comment:
Maybe you can use `PyModule_AddType()` in here.

Reply:
@shihai1991 yes sure.  There's some ugliness in this method relating to the type (it clears the tp_new method pointer for no reason).  I can try to clean this up.